scene/main/window.cpp

```diff
@@ -649,12 +651,52 @@ void Window::_event_callback(DisplayServer::WindowEvent p_event) {
 		case DisplayServer::WINDOW_EVENT_FOCUS_IN: {
 			focused = true;
 			_propagate_window_notification(this, NOTIFICATION_WM_WINDOW_FOCUS_IN);
+
+			if (!Engine::get_singleton()->is_editor_hint()) {
+				const int max_fps_when_unfocused = GLOBAL_GET("application/run/max_fps_when_unfocused");
+				const bool low_processor_mode_when_unfocused = GLOBAL_GET("application/run/low_processor_mode_when_unfocused");
+				if (max_fps_when_unfocused >= 1 || low_processor_mode_when_unfocused) {
+					if (OS::get_singleton()->is_stdout_verbose()) {
+						if (max_fps_when_unfocused >= 1 && low_processor_mode_when_unfocused) {
+							print_line(vformat("Window: Restoring previous maximum FPS (%d) and low-processor mode status (%s) on focus restore.", max_fps_on_focus_loss_previous, low_processor_mode_on_focus_loss_previous ? "enabled" : "disabled"));
```

Comment:
`print_verbose` might be better?

Reply:
It's already encapsulated in `is_stdout_verbose()`. I did this to prevent conditions from being evaluated for no reason.